main_logic/tts_client.py

```
@@ -1202,6 +1202,237 @@ async def async_worker():
+    if base_url.startswith('http://'):
+        ws_base = 'ws://' + base_url[7:]
+    elif base_url.startswith('https://'):
+        ws_base = 'wss://' + base_url[8:]
+    elif base_url.startswith('ws://') or base_url.startswith('wss://'):
+        ws_base = base_url
+    else:
+        ws_base = 'ws://' + base_url
+
+    WS_URL = f'{ws_base}/api/v3/tts/stream-input'
+
+    # 解析 voice_id：支持 "voice_id" 或 "voice_id|{JSON高级参数}" 格式
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)